libraries/tock-tbf/src/types.rs

```diff
@@ -130,6 +130,7 @@ pub enum TbfHeaderTypes {
     TbfHeaderStoragePermissions = 7,
     TbfHeaderKernelVersion = 8,
     TbfHeaderProgram = 9,
+    TbfHeaderShortId = 10,
```

Comment:
The book should be updated as well

Reply:
https://github.com/tock/book/pull/40 ?

Reply:
Ah, it just wasn't linked in the PR description